tests/e2e/tests/content-releases/releases-page.spec.ts

```diff
@@ -136,10 +134,9 @@ describeOnCondition(edition === 'EE')('Releases page', () => {
     });
 
     await test.step('releases should be updated in the release column of list view', async () => {
-      const releaseColumn = page.getByRole('button', { name: '2 releases' });
+      const releaseColumn = page.getByRole('button', { name: '1 release' });
       await releaseColumn.first().click();
       await expect(page.getByText('The Diamond Dogs')).toBeVisible();
-      await expect(page.getByText('Trent Crimm: The Independent')).toBeVisible();
     });
```

Comment:
Just curious why we only check for 1 now instead of 2? I can't remember but I think before the data set seeded the database with an initial release...

Reply:
Indeed there are two releases in the initial data, but the entry is only added into one of them in this e2e test, so I assume it was added into two of them when this test was created